components/widgets/menu/menu.jsx

```diff
@@ -29,15 +30,19 @@ export default class Menu extends React.PureComponent {
     }
 
     render() {
-        const {children, openUp, openLeft, id, ariaLabel} = this.props;
-        const styles = {};
-        if (openLeft && !isMobile()) {
-            styles.left = 'inherit';
-            styles.right = 0;
-        }
-        if (openUp && !isMobile()) {
-            styles.bottom = '100%';
-            styles.top = 'auto';
+        const {children, openUp, openLeft, id, ariaLabel, customStyles} = this.props;
+        let styles = {};
+        if (customStyles) {
+            styles = customStyles;
+        } else {
```

Comment:
Does this need to be in an else block, or would it make sense to merge those styles with the ones that are passed in? Something like
```javascript
let styles = customStyles ? {...customStyles} : {};

if (openLeft && !isMobile()) {
    styles.left = 'inherit';
}
```

Reply:
Same as the comment above, I wanted to replace the styles completely in this one instance to avoid breaking in other instances. Maybe you are right, not sure. But again, trying to avoid adverse side affects/interaction with other instances.